tests/fixtures/server.crt

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I'm wondering if there's a way to generate the key/cert on the fly.

Reply:
Yes, but it'd involve calling `openssl` from the command line and would increase the test suite runtime

Reply:
Alrighty.